src/new-client/src/app/lib/services/appointmentsService.ts

```diff
@@ -298,3 +302,21 @@ export const setSiteInformationForCitizen = async (
   handleEmptyResponse(response);
   revalidatePath(`/site/${site}/details`);
 };
+
+export const fetchAvailability = async (payload: FetchAvailabilityRequest) => {
```

Comment:
Might be okay for now, but it's worth noting that NextJS' only automatically memoizes GET requests because usually a POST would cause changes. If we ever find ourselves calling this more than once in a component tree we should consider adding in a `cache()` call

https://nextjs.org/docs/app/building-your-application/caching#request-memoization

https://nextjs.org/docs/app/building-your-application/caching#react-cache-function